### PR TITLE
Add kr resolution

### PR DESCRIPTION
**Pull request by ausonandres**
The purpose of this PR is to add functions in order to compute the energy resolution of the krypton peak and its error. Then, this information will be computed for each time bin and stored in time_evolution table inside maps.

### CODE REVIEW

File: krcal/core/kr_parevol_functions.py [hunk 1]
```diff
@@ -36,6 +42,50 @@ def get_number_of_time_bins(nStimeprofile: int,
...
+    Computes energy resolution of krypton peak.
+    Parameters
+    ----------
+    E: np.array
+        Array with kdst corrected energy.
+    Returns
+    -------
+        Relative energy resolution and error.
+    """
+    Ebins = 500
```
Comment (bpalmeiro) [inline krcal/core/kr_parevol_functions.py]: This parameter is hardcoded, should it be this way?

File: krcal/core/kr_parevol_functions.py [hunk 2]
```diff
@@ -36,6 +42,50 @@ def get_number_of_time_bins(nStimeprofile: int,
...
+    y, x  = np.histogram(E, Ebins)
+    x     = shift_to_bin_centers(x)
+    xu    = np.diff(x)[0] * 0.5
```
Comment (bpalmeiro) [inline krcal/core/kr_parevol_functions.py]: xu: this variable is not used

File: krcal/core/kr_parevol_functions.py [hunk 3]
```diff
@@ -36,6 +42,50 @@ def get_number_of_time_bins(nStimeprofile: int,
...
+    Epeak = np.average( x            , weights=y)
+    Speak = np.average((x - Epeak)**2, weights=y)**0.5
+
+    gauss_seed    = (      Npeak,     Epeak,  Speak)
+    gauss_lbounds = (          0,         0,      0)
+    gauss_ubounds = (1000*E.size, np.max(E), np.inf)
+
+    bkg_seed    = (      -1,       1,       1)
+    bkg_lbounds = (-np.inf, -np.inf, -np.inf)
+    bkg_ubounds = (      0, +np.inf, +np.inf)
```
Comment (bpalmeiro) [inline krcal/core/kr_parevol_functions.py]: Are we sure we want all these parameters also hardcoded?

**Comment by bpalmeiro** [comment]
Nice work! I would suggest adding a test for this new function.

File: krcal/core/kr_parevol_functions.py (hunk 2 above)
Comment (ausonandres) [reply krcal/core/kr_parevol_functions.py]: that's true, i'll remove it.

File: krcal/core/kr_parevol_functions.py (hunk 1 above)
Comment (ausonandres) [reply krcal/core/kr_parevol_functions.py]: ok, now Ebins will be an optional input of the function.

**Review by bpalmeiro — APPROVED** [review]
Easy and clean. Nice work!

**Pull request merged**